src/v2/Apps/Authentication/Server/redirectPostAuth.ts

```diff
@@ -0,0 +1,24 @@
+export function redirectPostAuth({ req, res }) {
```

Comment:
This function handles the logic to determine whether or not a redirect from artsy.net is "safe".

First, we ensure that all of the configured allowed hosts are *.artsy.net domains (or localhost is OK, if you're in local development).

Then, we redirect to either the artsy.net homepage (if the URL is not allowed), or to the requested external URL.